charts/sn-platform/templates/streamnative-console/streamnative-console-initialize.yaml

```diff
@@ -60,9 +60,17 @@ spec:
         {{- end }}
         - name: STREAMNATIVE_CONSOLE_BACKEND_URL
           value: http://{{ template "pulsar.streamnative_console.service" . }}:7750
+        {{- if not .Values.auth.vault.enabled }}
+        - name: SUPER_USER_NAME
+          value: admin
+        - name: SUPER_USER_PASSWORD
```

Comment:
When vault auth is not enabled, we can initialize the environment with an arbitrary account and password